mono_repo/lib/src/utils.dart

```diff
@@ -12,24 +12,25 @@ import 'mono_config.dart';
 import 'package_config.dart';
 import 'user_exception.dart';
 
-final packageConfigFileName = 'mono_repo.yaml';
+final rootConfigFileName = 'mono_repo.yaml';
+const _legacyConfigFileName = '.mono_repo.yml';
```

Comment:
nit: _legacyPkgConfigFileName

so it isn't confused with the root one that looks very similar (and is right above)

Reply:
ack!